node/src/main/scala/org/bitcoins/node/networking/P2PClient.scala

```diff
@@ -182,7 +202,7 @@ case class P2PClientActor(
       context watch connection
       val _ = handleEvent(event, proxy, ByteVector.empty)
     case Terminated(actor) if actor == proxy =>
-      context stop self
+      context.stop(self)
```

Comment:
same here

Reply:
It makes more sense to handle it in `unhandled()` so that it will be reconnecting  in any state:

```scala
  override def unhandled(message: Any): Unit = message match {
    case Terminated(actor) if actor == connection =>
       ...
```